_build_cfg.yml

```diff
@@ -615,8 +615,8 @@ Topics:
     Topics:
       - Name: Overview
         File: index
-      - Name: Using S2I images
-        File: customization
+      - Name: Customizing S2I images
```

Comment:
@ahardin-rh Just FYI, I went ahead and did this via https://github.com/openshift/openshift-docs/pull/3524.